HelpSource/Classes/Compander.schelp

```diff
@@ -14,113 +15,150 @@ argument::control
 
 argument::slopeAbove
-Same thing, but above the threshold. Values < 1.0 achieve compression (louder signals are attenuated); > 1.0, you get expansion (louder signals are made even louder). For 3:1 compression, you would use a value of 1/3 here.
+Slope of applied amplitude curve if control signal amplitude is above code::thresh::.
+Values < 1.0 achieve compression (louder signals are attenuated) whereas values > 1.0 expand (louder signals are made even louder). A value of code::1/3:: achieves a 3:1 compression.
 
 argument::clampTime
-The amount of time it takes for the amplitude adjustment to kick in fully. This is usually pretty small, not much more than 10 milliseconds (the default value). I often set it as low as 2 milliseconds (0.002).
+time it takes for the amplitude adjustment to kick in fully after the control signal is above code::thresh::. Usually a small value around 10ms (0.01), often set as low as 2ms (0.002).
 
 argument::relaxTime
-The amount of time for the amplitude adjustment to be released. Usually a bit longer than clampTime; if both times are too short, you can get some (possibly unwanted) artifacts.
+time for the amplitude adjustment to be released (control signal below code::thresh::). Usually longer than code::clampTime::; depending on the input and cntrol signal, setting both times too short results in (possibly unwanted) artifacts.
 
 argument::mul
-
+factor to multiply the UGen's output with.
 argument::add
+factor to add to the UGen's output.
 
 discussion::
-If any of this is confusing, see http://en.wikipedia.org/wiki/Audio_level_compression
+See for example http://en.wikipedia.org/wiki/Audio_level_compression for a more in-depth explanation.
 
 examples::
 
 code::
 (
-// example signal to process
-play({
-    var z;
-    z = Decay2.ar(
-        Impulse.ar(8, 0,LFSaw.kr(0.3, 0, -0.3, 0.3)),
-        0.001, 0.3, Mix.ar(Pulse.ar([80,81], 0.3)))
-})
+// signal (clean)
+x = {
+    Decay2.ar(
+		Impulse.ar(8, 0, LFSaw.kr(0.3, 1, -0.3, 0.3)),
+		0.001,
+		0.3
+	)
+	* Mix.ar(Pulse.ar([80,81], 0.3))
+}.play;
 )
-::
 
-code::
+x.free;
+
 (
 // noise gate
-play({
+x = {
     var z;
-    z = Decay2.ar(
-        Impulse.ar(8, 0,LFSaw.kr(0.3, 0, -0.3, 0.3)),
-        0.001, 0.3, Mix.ar(Pulse.ar([80,81], 0.3)));
-    Compander.ar(z, z,
-        thresh: MouseX.kr(0.1, 1),
+
+	// signal (clean)
+	z = Decay2.ar(
+		Impulse.ar(8, 0, LFSaw.kr(0.3, 1, -0.3, 0.3)),
+		0.001,
+		0.3
+	)
+	* Mix.ar(Pulse.ar([80,81], 0.3));
+
+	// apply gate (mouse x sets treshold)
+	Compander.ar(z, z,
+        thresh: MouseX.kr(0.001, 1),
         slopeBelow: 10,
         slopeAbove: 1,
         clampTime: 0.01,
         relaxTime: 0.01
     );
```

Comment:
leave out trailing semicolon, if you like. the compander is intended as the return value of the function, and this conveys that fact better.